src/ol_infrastructure/applications/tika/__main__.py

```diff
@@ -173,6 +173,13 @@
     "x_access_token"
 ]
 
+# Store the access token in vault
+vault.generic.Secret(
+    "tika-server-x-access-token-vault-secret",
+    path="secret-operations/tika/access-token",
+    data_json=f'{{"value": "{x_access_token}"}}',
```

Comment:
Why not do this as `json.dumps({"value": x_access_token})`?

Reply:
🤷